lib/Sema/DerivedConformanceDifferentiable.cpp

```diff
@@ -489,6 +489,7 @@ getOrSynthesizeTangentVectorStruct(DerivedConformance &derived, Identifier id) {
     newMember->copyFormalAccessFrom(member, /*sourceIsParentContext*/ true);
     newMember->setValidationToChecked();
     newMember->setSetterAccess(member->getFormalAccess());
+    newMember->getSynthesizedAccessor(AccessorKind::Get);
```

Comment:
This should not be necessary. Is there a place we're using getAccessor() instead of getOpaqueAccessor()?

Reply:
I think you're right. I was working around some logic that was using the fact that ::Get was synthesized to detect what kind of var something was.